### ISSUE TITLE
mark points obtained by converting to marker are not animated

### ISSUE THREAD

**Issue by ShuJiangLiu** [david-cattermole/mayaMatchMoveSolver#249]
### Problem or Feature
Hello, I can see in the camera that the locator points I have tracked perfectly match the head, but when I use the locator to generate mkr points by covert to marker, I can see in the camera that only the last frame has been picked up and no other frame range has been picked up And it turns out that his translateX and Y values haven't changed at all

**Expected behavior:**

**Actual behavior:**

### Steps to Reproduce

1. [First Step]
2. [Second Step]
3. [and so on...]

### Software Versions

- mmSolver version: 0.4.6

- Maya version:2022.3

Operating System: Windows 10 Pro v2009 (Build: 19045)
Cut ID: 202110272215-ad32f8f1e6
Maya Version: Autodesk MAYA 2022.3
Maya API: 20220300
Qt Version: 5.15.2
Compositing Manager: True
Window Manager: Windows Aero
GPU:   Version : 2016.11.62.12. Feature Level 5.
  Adapter : NVIDIA GeForce RTX 4070 Ti
  Vendor ID: 4318. Device ID : 10114
  Driver : 袀栿ǐ:17.1.58.818.
  API : DirectX V.11.
  Max texture size : 16384 * 16384.
  Max texture coords : 32
  Shader versions supported (Vertex: 5, Geometry: 5, Pixel 5).
  Active stereo support available : 0
  GPU Memory Limit : 12282 MB.
  CPU Memory Limit : 124298 MB.

- Operating System (OS): [Possible options: Windows or LinuxOS]

**Comment by david-cattermole** [comment]
Hi @ShuJiangLiu,

Thank you for reporting this issue.

I seem to be getting the following error when I try to use the menu "mmSolver > Convert To Marker":
```
# Traceback (most recent call last):
#   File "C:\Users\username\Documents\maya\2022\modules\mayaMatchMoveSolver-0.4.6-maya2022-win64\python\mmSolver\tools\convertmarker\tool.py", line 147, in main
#     delete_static_anim_curves,
#   File "C:\Users\username\Documents\maya\2022\modules\mayaMatchMoveSolver-0.4.6-maya2022-win64\python\mmSolver\tools\convertmarker\lib.py", line 166, in create_markers_from_transforms
#     end_frame,
#   File "C:\Users\username\Documents\maya\2022\modules\mayaMatchMoveSolver-0.4.6-maya2022-win64\python\mmSolver\tools\convertmarker\lib.py", line 56, in convert_nodes_to_marker_data_list
#     asNormalizedCoordinate=True,
#   File "<string>", line 2, in mmReprojection
# RuntimeError: Maya command error
```
This error is probably related to your issue - either way, it needs to be fixed.

**Comment by david-cattermole** [comment]
@ShuJiangLiu has confirmed (through email) that the error above is the same as he gets, however upon investigation even after fixing that error, there appears to be an error in the Convert to Marker UI. 
The default start frame value is being used when the UI is set to use the "Timeline (Inner)" frame range.

I have found that it's possible to fix the problem by manually entering the Frame Range as "Custom", and typing the Start/End frame numbers.
![image](https://github.com/david-cattermole/mayaMatchMoveSolver/assets/891288/1581ae7a-a416-4250-86ff-0e3f5540f33a)

This is a secondary bug

**Comment by david-cattermole** [comment]
I've fixed this issue and it will be released in v0.4.7, which will hopefully be released soon.

**Comment by david-cattermole** [comment]
Hi @ShuJiangLiu, v0.4.7 has now been released with the fix included.

https://github.com/david-cattermole/mayaMatchMoveSolver/releases/tag/v0.4.7
